modules/javafx.controls/src/main/java/javafx/scene/control/skin/SliderSkin.java

```diff
@@ -343,6 +343,7 @@ public Object queryAccessibleAttribute(AccessibleAttribute attribute, Object...
         };
         thumb.getStyleClass().setAll("thumb");
         thumb.setAccessibleRole(AccessibleRole.THUMB);
+        dragStart = new Point2D(thumb.getLayoutX(), thumb.getLayoutY());
```

Comment:
`dragStart` should be the exact mouse position where mouse pressed has occurred.
With this change mouse gets moved to the layout X, Y position of thumb, even though the mouse press occurred at any other location inside the thumb.

Reply:
This change makes sure that `dragStart` is not null when `setOnMouseDragged `event handler is called.
As suggested in the issue, I tried to initialize `dragStart` in `layoutChildren` method but it was causing side effects.
I checked for other event handlers such as `onDragDetected` and `onMouseDragEntered`, but none of these event handlers were invoked. Looks like these events are also consumed by tooltip similar to how it consumes `setOnMousePressed`.
I haven't seen above mentioned side effect while checking the fix. I will check again.